stock_delivery_report_credentials/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Author: Andrius Laukavičius. Copyright: JSC Focusate.
+# See LICENSE file for full copyright and licensing details.
+{
+    'name': "Delivery Slip Report Credentials",
+    'version': '14.0.2.0.0-rc.1',
```

Comment:
Isn't this module new on v14?

Reply:
It was old, then was split up. So yes now new:)